lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -656,6 +710,114 @@ def self.type_from_v1type(v1type)
       V1_TYPE_MAP[v1type]
     end
 
+    def self.to_manifest_hash(iTrigger)
+      # TODO: remove these
+      # v1trigger = V1.from_iTrigger(iTrigger)
+      # return V1.to_manifest_hash(v1trigger)
+
+      # require 'pry'; binding.pry
+      if V1_TYPE_MAP.key(iTrigger.Type).nil?
```

Comment:
As this is in V2...Should this actually error if it doesn't exist in the v1 map?